packages/core/src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -136,7 +136,7 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
     private async _loadPaymentMethod(gatewayId: string, methodId: string, options?: RequestOptions): Promise<InternalCheckoutSelectors> {
         try {
             return await this._store.dispatch(
-                this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}?method=${methodId}`, options)
+                this._paymentMethodActionCreator.loadPaymentMethod(gatewayId, {...options, params: {method: methodId}})
```

Comment:
@josesonora As discussed, could you review afterpay strategy when you have time? Thanks